public/app/app.ts

```diff
@@ -114,16 +114,9 @@ export class GrafanaApp {
 
       // disable tool tip animation
       $.fn.tooltip.defaults.animation = false;
-
       this.angularApp.init();
 
-      // Preload selected app plugins
-      const promises: Array<Promise<any>> = [];
-      for (const modulePath of config.pluginsToPreload) {
-        promises.push(importPluginModule(modulePath));
-      }
-
-      await Promise.all(promises);
+      await preloadPlugins(config.pluginsToPreload);
```

Comment:
Still feels wrong to wait booting the app for the plugins to load but could be some future case that does require that, but not sure we have one right now

Reply:
Agree - I will have a look at that in a separate PR.

Reply:
I need to verify that it doesn't impact enterprise with the body and header hooks.

Reply:
I might be misreading the code but I don't think the pageBanners hook is an issue. Looks like it's the `<Banner />` component that plugins communicate with via `appEvents` rather than creating Banners themselves. 🤔

Reply:
Ok ok, cool!